### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by muga**

### CODE REVIEW

File: src/main/java/com/treasuredata/api/TdApiClientConfig.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import com.google.common.base.Optional;
 
 public class TdApiClientConfig
-        implements TdApiConstants
+        extends TdApiConstants
```
Comment (nahi) [inline src/main/java/com/treasuredata/api/TdApiClientConfig.java]: Do we need to implement/extend TdApiConstants? In general manner Constants is public final class with private default constructor, or interface. Any manner we can choose but I don't understand the reason we do subclassing.

**Comment by nahi** [comment]
Hmm, those 4 commits look safe as long as:
* code formatting change only includes code formatting change (you just did it by IDE's code formatter)
* refactoring does only refactoring (does not change code behavior, does not include bugfixes)
I mean that https://github.com/treasure-data/embulk-output-td/commit/85b5626f026c1dbb8dc36e15dd99386a9abe5c3d is safe because it does not change main code and test passes. :-)

Next time you can do the same thing in this order:
1. Write test as extent possible without changing code
2. Do refactoring
3. Change code formatting in different PR.

File: src/main/java/com/treasuredata/api/TdApiClientConfig.java (hunk 1 above)
Comment (muga) [inline src/main/java/com/treasuredata/api/TdApiClientConfig.java]: Thank you for letting me know. use public final class for `TdApiConstants` 3189e41b76018ea1d697d7723ab028e0f02d1915

**Comment by muga** [comment]
> 1. Write test as extent possible without changing code
> 2. Do refactoring
> 3. Change code formatting in different PR.

Thanks. I understand. To add unit tests, I needed to do some refactoring within target classes. Next time, I will include such refactoring commits in testing commits. Other refactoring should be different PR. code formatting as well.

**Comment by nahi** [comment]
LGTM!

**Pull request merged**